write-fonts/src/tables/glyf.rs

```diff
@@ -57,18 +59,11 @@ impl Glyph {
     /// The bounding box for the glyph
     pub fn bbox(&self) -> Bbox {
         match self {
+            Glyph::Empty => todo!(),
```

Comment:
Hmm... this is a big footgun. Un-lgtm!

Reply:
ahaha oops. This is actually something to discuss, and maybe @dfrg has input? Basically the bounding box of an empty glyph is *not* (0,0), (0,0), because that implies that the point (0,0) is contained. I think for correctness this should probably return an Option? This has been a source of bugs in other geometry code I've written in the past.

Reply:
I think returning an option here makes sense. We can always unwrap_or_default if we want the all zero bbox for some reason.